config/sync/core.extension.yml

```diff
@@ -9,6 +9,7 @@ module:
   editor: 0
   entity_reference_revisions: 0
   field: 0
+  field_ui: 0
```

Comment:
The `field_ui` module should only be used during development, i.e. it should not be enabled in config.